images/kubemci/Makefile

```diff
@@ -0,0 +1,35 @@
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+VERSION = $(shell date +v%Y%m%d)-$(shell git describe --tags --always --dirty)
+
+IMG = "gcr.io/k8s-testimages/e2e-kubemci"
+
+# Build a versioned image.
+image:
+	docker build --no-cache -t "$(IMG):$(VERSION)" --build-arg "IMAGE_ARG=$(IMG):$(VERSION)" .
+
+# Build both the versioned and latest images.
+image-latest: image
+	docker build --no-cache -t "$(IMG):latest" --build-arg "IMAGE_ARG=$(IMG):latest" .
+
+# Build and push the versioned image.
+push: image
+	gcloud docker -- push "$(IMG):$(VERSION)"
```

Comment:
oh, one more thing, see https://github.com/kubernetes/test-infra/pull/6765, you probably should just use docker push here

Reply:
done